Cargo.toml

```diff
@@ -24,4 +14,13 @@ panic = "abort"
 panic = "abort"
 
 [workspace.dependencies]
-salty.version = "0.3"
+anyhow = { version = "1", default-features = false }
+camino = { version = "1.1.7" }
+dice-verifier = { path = "../dice-util/verifier" }
```

Comment:
This should use a proper remote dep. This is pending a review by @flihp: https://github.com/oxidecomputer/dice-util/pull/217